tests/Nox.Types.Extensions.Tests/Types/LanguageCode/LanguageExtensionsTests.cs

```diff
@@ -0,0 +1,22 @@
+using FluentAssertions;
+using Nox.Reference;
+
+namespace Nox.Types.Extensions.Tests.Types.LanguageCode;
+
+public class LanguageExtensionsTests : WorldTestBase
+{
+    [Fact]
+    public void GetReferenceLanguageCode_ValidIsoLanguage_ReturnsExpectedLanguageCode()
+    {
```

Comment:
Should split this into two tests, since were are asserting two scenarios: when we have iso code and when we don't have it?

Reply:
Desired changes have been applied